torch/csrc/jit/ivalue.h

```diff
@@ -101,13 +101,18 @@ struct ConstantString : at::Retainable {
   }
 };
 
+struct World {
+  World() : world_id(1) {}
+  int64_t world_id;
```

Comment:
Nothing, it's just a placeholder, since empty structs are not allowed. In the future, we could put basic debugging info in there, or thread local state, but I couldn't thinking of anything useful so far.